rosys/hardware/gnss.py

```diff
@@ -217,10 +240,11 @@ def _find_device(self) -> str:
                 return port.device
         raise RuntimeError('No GNSS device found')
 
-    def _connect_to_device(self, port: str, *, baudrate: int = 115200, timeout: float = 0.2) -> serial.Serial:
+    def _connect_to_device(self, port: str, *, baudrate: int = 460800, timeout: float = 0.2) -> serial.Serial:
         self.log.debug('Connecting to GNSS device "%s"...', port)
         try:
-            return serial.Serial(port=port, baudrate=baudrate, timeout=timeout)
+            serial_connection = serial.Serial(port=port, baudrate=baudrate, timeout=timeout)
+            return serial_connection
```

Comment:
I think we can revert this change:
```suggestion
            return serial.Serial(port=port, baudrate=baudrate, timeout=timeout)
```